### PR TITLE
[zh-cn] sync service.md

### PR DESCRIPTION
**Pull request by Eason1118**
Keep it consistent with the original text and optimize the Chinese description
Original link: [#47938](https://github.com/kubernetes/website/pull/47938)
please review

### CODE REVIEW

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 1]
```diff
@@ -1220,23 +1220,18 @@ Unprefixed names are reserved for end-users.
 没有前缀的名字是保留给最终用户的。
 
 <!--
-#### Specifying IPMode of load balancer status {#load-balancer-ip-mode}
+#### Load balancer IP address mode {#load-balancer-ip-mode}
 -->
-#### 指定负载均衡器状态的 IPMode    {#load-balancer-ip-mode}
+#### 负载均衡器IP地址模式    {#load-balancer-ip-mode}
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:1225]:
```suggestion
#### 负载均衡器 IP 地址模式    {#load-balancer-ip-mode}
```

File: content/zh-cn/docs/concepts/services-networking/service.md [hunk 2]
```diff
@@ -1220,23 +1220,18 @@ Unprefixed names are reserved for end-users.
...
-named `LoadBalancerIPMode` allows you to set the `.status.loadBalancer.ingress.ipMode` 
-for a Service with `type` set to `LoadBalancer`. 
+For a Service of `type: LoadBalancer`, a controller can set `.status.loadBalancer.ingress.ipMode`. 
 The `.status.loadBalancer.ingress.ipMode` specifies how the load-balancer IP behaves. 
 It may be specified only when the `.status.loadBalancer.ingress.ip` field is also specified.
 -->
-作为 Kubernetes 1.30 中的 Beta 级别特性，通过名为 `LoadBalancerIPMode`
-的[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)允许你为
-`type` 为 `LoadBalancer` 的服务设置 `.status.loadBalancer.ingress.ipMode`。
+对于 `type: LoadBalancer` 的服务，控制器可以设置`.status.loadBalancer.ingress.ipMode`。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/service.md:1234]:
```suggestion
对于 `type: LoadBalancer` 的 Service，控制器可以设置 `.status.loadBalancer.ingress.ipMode`。
```

**Comment by Eason1118** [comment]
Thanks. have a nice weekend

**Pull request merged**